Example/ViewController.swift

```diff
@@ -412,6 +412,7 @@ extension ViewController: NavigationViewControllerDelegate {
         
         //If we're not in a "Multiple Stops" demo, show the normal EORVC
         if navigationViewController.navigationService.router.routeProgress.isFinalLeg {
+            endCarPlayNavigation(canceled: false)
```

Comment:
Shouldn’t `navigationViewControllerDidDismiss(_:byCanceling:)` get called in this situation?

Reply:
[It does.](https://github.com/mapbox/mapbox-navigation-ios/blob/master/MapboxNavigation/CarPlayNavigationViewController.swift#L189-L199)